BusyTiming/HomePage/TimerModel.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 // ***** I kept it 'TimerModel' for now to not confuse you as you are familiar with this name.
 // ***** Later it should be renamed to something like SavedTimePointRepository
-class TimerModel {
+class LastTimePoint {
```

Comment:
It's a best practice to provide classes that store something with names that end with something like 'Storage' or 'Repository'. In other words, the name (of the class or a variable or whatever else) should reflect what the class actually DOES. Name "LastTimePoint" looks like a name of a data object that just represents the time point. If the class actually STORES such points it should be named appropriately.

Reply:
BTW, if you renamed the class, you can remove the comment above then.